### PR TITLE
Treat season as decimal to make address form

### PR DESCRIPTION
**Pull request by U-lis**
This is what I originally intended.
No alphabet in decimal, so no capital-small character problem.

### CODE REVIEW

**Comment by U-lis** [comment]
This seems like #2647, but this one is using decimal, not hexadecimal.
One of two PRs must be ignored.

File: .Lib9c.Tests/Helper/AdventureBossHelperTest.cs [hunk 1]
```diff
@@ -17,6 +17,16 @@ public class AdventureBossHelperTest
         private Address _avatarAddress = new PrivateKey().Address;
         private string _name = "wanted";
 
+        [Theory]
+        [InlineData(1, "0000000000000000000000000000000000000001")]
+        [InlineData(10, "0000000000000000000000000000000000000010")]
+        [InlineData(16, "0000000000000000000000000000000000000016")]
+        [InlineData(100, "0000000000000000000000000000000000000100")]
```
Comment (ipdae) [inline .Lib9c.Tests/Helper/AdventureBossHelperTest.cs:24]: 혹시 모르니.. 이거 단위를 더 큰 값으로도 테스트해볼수있을까요?

Comment (ipdae) [reply .Lib9c.Tests/Helper/AdventureBossHelperTest.cs:24]: 아.. 이게 decimal format이군요 별문제 없을것같네요.

**Review by ipdae — APPROVED** [review]

**Comment by ipdae** [comment]
충돌이 납니다.

**Pull request merged**